### PR TITLE
Fix travis ci error and change coverage platform to codecov

### PR DESCRIPTION
**Pull request by lovepoem**
Change ./mvn to ./mvnw  and change coverage platform to codecov

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <version>3.0.0-beta-SNAPSHOT</version>
     <packaging>pom</packaging>
     <name>${project.artifactId}</name>
-
```
Comment (TeslaCN) [inline pom.xml]: Keep indents (including blank lines) consistent with the previous one.
Ref https://shardingsphere.apache.org/community/en/contribute/code-conduct/

**Review by TeslaCN — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+install: true
+
+cache:
+  directories:
+    - "$HOME/.m2"
+
 before_script:
-    - echo "MAVEN_OPTS='-Xmx1024m -XX:MaxPermSize=256m'" > ~/.mavenrc
+  - echo "MAVEN_OPTS='-Xmx1024m -XX:MaxMetaspaceSize=256m'" > ~/.mavenrc
+  - rm -rf $HOME/.m2/repository/org/apache/shardingsphere/elasticjob
```
Comment (terrymanu) [inline .travis.yml]: Why rm folder in before script, can `mvn clean install` do it?

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (lovepoem) [reply .travis.yml]: Removed useless code

Comment (terrymanu) [reply .travis.yml]: It seems like still existed.

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (lovepoem) [reply .travis.yml]: Finished

**Review by terrymanu — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -464,13 +464,6 @@
                 <version>${cobertura-maven-plugin.version}</version>
                 <configuration>
                     <check>
-                        <branchRate>80</branchRate>
-                        <lineRate>80</lineRate>
-                        <haltOnFailure>true</haltOnFailure>
-                        <totalBranchRate>80</totalBranchRate>
-                        <totalLineRate>80</totalLineRate>
-                        <packageLineRate>80</packageLineRate>
-                        <packageBranchRate>80</packageBranchRate>
```
Comment (terrymanu) [inline pom.xml]: We should not remove this

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (lovepoem) [reply pom.xml]: Reverted pom.xml

**Review by terrymanu — APPROVED** [review]

**Pull request merged**